packages/@okta/vuepress-theme-prose/components/FrontPageCodeMirror.vue

```diff
@@ -55,7 +64,7 @@ widget.renderEl({
     destroyCm() {
       const element = this.codemirror.doc.cm.getWrapperElement();
       element && element.remove && element.remove();
-    },
-  },
+    }
+  }
```

Comment:
nit and not a blocker but i advocate for trailing `,` since it produces cleaner diffs. Consider turning it on in your prettier defaults

Reply:
Weird, `trailingOption` is set to `ES5` by default, which means it should have trailing commas for object and arrays but not function arguments^^

Reply:
```["INFO" - 4:19:32 PM] Detected local configuration (i.e. .prettierrc or .editorconfig), VS Code configuration will not be used```

Reply:
Hmm there is `.editorconfig` in the project root. I guess that's why.